pkg/cache/cluster_info.go

```diff
@@ -47,8 +48,8 @@ type ClusterInfo struct {
 	pendingSchedulerEvents chan interface{}
 
 	// RM Event Handler
-	EventHandlers handler.EventHandlers
-
+	EventHandlers    handler.EventHandlers
+	skipConfigUpdate bool
```

Comment:
based on the discussion, I don't think we need this flag anymore.
Let's keep the core side change as simple as possible

Reply:
You are right, I removed it